src/evergreen/build.py

```diff
@@ -81,6 +81,16 @@ def status_counts(self) -> StatusCounts:
         """Get the status counts of the build."""
         return StatusCounts(self.json["status_counts"], self._api)
 
+    def get_human_readable_project_id(self) -> str:
```

Comment:
Could we rename this to something like `get_project_identifier`? "Identifier" is the terminal that evergreen uses and it would be nice to stay consistent with that.